tests/functional/test_clinical_trials_parser_from_folder.py

```diff
@@ -6,7 +6,7 @@
 
 HERE = os.path.dirname(os.path.abspath(__file__))
```

Comment:
HERE?

Reply:
I don' know how should we name the path to the current file directory :D

Reply:
some ideas? :>

Reply:
current file directory sounds better

Reply:
changed, thanks :)